<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="LAPTOP-H0VOD1U4" failures="1" tests="1" name="Test" time="5.54" errors="0" timestamp="2025-03-04T21:30:34 GMT-05:00">
  <testcase classname="testscripts.LoginTest" name="verifyUserLoginWithValidUsernameAndInvalidPassword" time="1.454">
    <failure type="java.lang.AssertionError" message="Successful login with invalid password expected [Logi | 7rmart supermarket] but found [Login | 7rmart supermarket]">
      <![CDATA[java.lang.AssertionError: Successful login with invalid password expected [Logi | 7rmart supermarket] but found [Login | 7rmart supermarket]
at testscripts.LoginTest.verifyUserLoginWithValidUsernameAndInvalidPassword(LoginTest.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserLoginWithValidUsernameAndInvalidPassword -->
  <testcase classname="testscripts.LoginTest" name="verifyUserLoginWithInvalidUsernameAndValidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- verifyUserLoginWithInvalidUsernameAndValidPassword -->
  <testcase classname="testscripts.LoginTest" name="verifyUserLoginWithValidUsernameAndValidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- verifyUserLoginWithValidUsernameAndValidPassword -->
  <testcase classname="testscripts.LoginTest" name="verifyUserLoginWithInvalidUsernameAndInvalidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- verifyUserLoginWithInvalidUsernameAndInvalidPassword -->
</testsuite> <!-- Test -->
